package/yast2-ca-management.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Oct  6 10:03:34 UTC 2017 - locilka@suse.cz
+
+- Fix directory navigation when trying to go up one level (..)
```

Comment:
NP: `to go one level up` sounds probably better and I'd quote the double dots, it took me a while to get the meaning, i.e. `".."`.